compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -4245,7 +4245,7 @@ static void intCompareNarrower(TR::Node * node, TR::Simplifier * s, TR::ILOpCode
                {
                if (secondChild->getReferenceCount() > 1)
                   {
-                  TR::Node * newSecondChild = TR::Node::sconst(secondChild, (int16_t)secondChild->getInt());
+                  TR::Node * newSecondChild = TR::Node::cconst(secondChild, (int16_t)secondChild->getInt());
```

Comment:
Isn't this backward? Why do we need this to be `cconst` instead of `sconst`?

Reply:
Note that the `cconst` here is `Node::cconst()` instead of `TR::cconst`. `Node::cconst()` essentially creates `TR::sconst`, which does the same thing as in `Node::sconst()`. Therefore, this change is not wrong but i think it is better for me to remove `Node::cconst()` and map all its uses to `Node::sconst()`. Do you agree on this?

Reply:
Yes, I agree, that more sense.

Reply:
I have mapped all references of `Node::cconst()` to `Node::sconst()` in 14de6f2 but I did not remove `Node::cconst()` because there are still references to it in OpenJ9. We can delete this function in future PR (e.g. in https://github.com/eclipse/openj9/pull/6465)